### PR TITLE
Introduce `lhs` and `rhs` aliases to assignment nodes for consistency.

### PR DESCRIPTION
**Pull request by dvandersluis**
The methods are already available for `masgn`, so this will allow all assignment node types to have the same two methods so that they can be dealt with in unison. Once released I've found some places in rubocop where more destructuring can be simplified.

Follows https://github.com/rubocop/rubocop/pull/13413#discussion_r1827267164

I didn't add tests for these aliases but am happy to if desired!

### CODE REVIEW

File: lib/rubocop/ast/node/asgn_node.rb [hunk 1]
```diff
@@ -12,13 +12,15 @@ class AsgnNode < Node
...
       end
+      alias lhs name
 
       # The expression being assigned to the variable.
       #
       # @return [Node] the expression being assigned.
       def expression
         node_parts[1]
       end
+      alias rhs name
```
Comment (bquorning) [inline lib/rubocop/ast/node/asgn_node.rb:23]:
```suggestion
      alias rhs expression
```

Comment (dvandersluis) [reply lib/rubocop/ast/node/asgn_node.rb]: Oops thanks @bquorning! Updated.

**Comment by marcandre** [comment]
Looks good, thank you.

Released as 1.34.0.

Minor thing: If you think about it for upcoming PRs, please make a Changelog entry...

**Comment by dvandersluis** [comment]
Sorry @marcandre! I'll add a changelog for this right now.

**Pull request merged**